shell/browser/api/electron_api_base_window.cc

```diff
@@ -864,6 +864,12 @@ void BaseWindow::SetVibrancy(v8::Isolate* isolate, v8::Local<v8::Value> value) {
   window_->SetVibrancy(type);
 }
 
+void BaseWindow::SetBackgroundMaterial(v8::Isolate* isolate,
+                                       v8::Local<v8::Value> value) {
```

Comment:
Following should do the conversion,

```suggestion
void BaseWindow::SetBackgroundMaterial(v8::Isolate* isolate,
                                       const std::string& material_type) {
```